logpyle/__init__.py

```diff
@@ -385,7 +388,18 @@ def _set_up_schema(db_conn: Connection) -> int:
         lineno integer
         )""")
 
-    schema_version = 2
+    # schema_version < 3 does not have the logging table
+    db_conn.execute("""
+      create table logging (
+        rank integer,
```

Comment:
May want to store wall time? (Possibly also for warnings?)

Reply:
I added a unixtime column in [97add34](https://github.com/illinois-ceesd/logpyle/pull/94/commits/97add3405575f11dafd38f5a6ac53475ce2e0213) to both tables.